tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -301,10 +300,39 @@ internal static partial class ConfigurationKeys
         public const string KafkaCreateConsumerScopeEnabled = "DD_TRACE_KAFKA_CREATE_CONSUMER_SCOPE_ENABLED";
 
         /// <summary>
-        /// Configuration key for enabling or disabling CI Visibility.
-        /// Default is value is false (disabled).
+        /// String constants for CI Visibility configuration keys.
+        /// </summary>
+        public static class CIVisibility
+        {
+            /// <summary>
+            /// Configuration key for enabling or disabling CI Visibility.
+            /// Default is value is false (disabled).
+            /// </summary>
+            public const string Enabled = "DD_CIVISIBILITY_ENABLED";
+
+            /// <summary>
+            /// Configuration key for enabling or disabling Agentless in CI Visibility.
+            /// Default is value is false (disabled).
+            /// </summary>
+            public const string AgentlessEnabled = "DD_CIVISIBILITY_AGENTLESS_ENABLED";
+        }
+
+        /// <summary>
+        /// String constants for proxy configuration keys.
         /// </summary>
-        public const string CIVisibilityEnabled = "DD_CIVISIBILITY_ENABLED";
+        public static class Proxy
+        {
+            /// <summary>
+            /// Configuration key to set a proxy server for https requests.
+            /// </summary>
+            public const string ProxyHttps = "DD_PROXY_HTTPS";
+
+            /// <summary>
+            /// Configuration key to set a list of hosts that should bypass the proxy.
+            /// The list is space-separated.
+            /// </summary>
+            public const string ProxyNoProxy = "DD_PROXY_NO_PROXY";
+        }
```

Comment:
We should make these changes in a separate PR since they are not related to the sql obfuscator.

Reply:
Resolving this comment since it's not part of your changes.